releases/release-5.4.0.md

```diff
@@ -232,7 +232,7 @@ In v5.4, the key new features or improvements are as follows:
 
     In most cases, when executing SQL statements, the optimizer only uses statistics of some columns (such as columns in the `WHERE`, `JOIN`, `ORDER BY`, and `GROUP BY` statements). These used columns are called `PREDICATE COLUMNS`.
 
-    Since v5.4.0, you can set the value of the [`tidb_enable_column_tracking`](/system-variables.md#tidb_enable_column_tracking-new-in-v540) system variable to `ON` to enable TiDB to collect `PREDICATE COLUMNS`.
+    Since v5.4.0, you can set the value of the [`tidb_enable_column_tracking`](https://docs.pingcap.com/tidb/v5.4/system-variables#tidb_enable_column_tracking-new-in-v540) system variable to `ON` to enable TiDB to collect `PREDICATE COLUMNS`.
```

Comment:
Since this variable is not removed, we can just keep the link as it is.
```suggestion
    Since v5.4.0, you can set the value of the [`tidb_enable_column_tracking`](/system-variables.md#tidb_enable_column_tracking-new-in-v540) system variable to `ON` to enable TiDB to collect `PREDICATE COLUMNS`.
```